t/apicast-policy-chains.t

```diff
@@ -0,0 +1,65 @@
+use lib 't';
+use TestAPIcast 'no_plan';
+
+run_tests();
+
+__DATA__
+
+=== TEST 1: custom policy chain
+This test uses the phase logger policy to verify that all of its phases are run
+when we use a policy chain that contains it. The policy chain also contains the
+normal apicast policy, so we can check that the authorize flow continues
+working. Notice that 'post_action' and 'log' are not ran. There's no
+post_action defined and log only runs when post_action runs.
+Also, init and init_worker are run before these tests, so what's logged in them
```

Comment:
I didn't explain this well. init() and init_worker() do not run because I defined the custom policy as a local/service one instead of a global one. The service is not available during the init() and init_worker() phases, and that's why local/service policies cannot run in init() and init_worker().